Makefile

```diff
@@ -34,3 +34,13 @@ test:
 cbindgen:
 	cbindgen glean-core/ffi --lockfile Cargo.lock -o glean-core/ffi/examples/glean.h
 .PHONY: cbindgen
+
+docs:
+	bin/build-docs.sh
```

Comment:
currently `build-docs.sh` builds both the book and the Rust docs.
We should therefore either rename the file to reflect this or move the kotlin doc generation into it.

Reply:
Ah, we build the kotlin docs in an independent CI step, therefore merging this into a single script is hard.

What do you think about a file name reflecting this? `build-rust-docs.sh`

Reply:
Yeah, that rename makes sense.  We could also probably build all docs in the same Android CI image, but that could get harder once we add Swift etc...  So let's just rename this for now.